pkg/util/utils.go

```diff
@@ -71,7 +69,7 @@ func Retry(attempts int, callback func() error) (err error) {
 	return RetryAfter(attempts, callback, 0)
 }
 
-func ValidateProxyURI(uri string) bool {
+func ValidProxyURI(uri string) bool {
 	return govalidator.IsURL(uri)
```

Comment:
Proxy settings should 'always' specify a port, as there is no real default and we can not assume this; IsURL will not enforce this.